README.adoc

```diff
@@ -0,0 +1,202 @@
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+:toc:
+:toc-placement!:
+
+= Mobile Security Service
+
+:toc:
+toc::[]
+
+== Overview
+
+This is the server component of the AeroGear Mobile Security Service. It is a RESTful API that allows developers to view, enable and disable specific versions of applications on demand, with the information stored in a PostgreSQL database.
+
+* https://circleci.com/gh/aerogear/mobile-security-service.svg?style=svg[CircleCI]
```

Comment:
These need to be displayed at the top of the Readme and in badge format. This should do the trick (I haven't tested it though)

image:{https://circleci.com/gh/aerogear/mobile-security-service.svg?style=svg}[CircleCI,link={https://circleci.com/gh/aerogear/mobile-security-service}]
image:{https://goreportcard.com/badge/github.com/aerogear/mobile-security-service}[Go Report Card,link={https://goreportcard.com/badge/github.com/aerogear/mobile-security-service}]
image:{https://coveralls.io/repos/github/aerogear/mobile-security-service/badge.svg?branch=master}[Coverage Status,link={https://coveralls.io/github/aerogear/mobile-security-service?branch=master}]
image:{https://img.shields.io/:license-Apache2-blue.svg}[License,link={http://www.apache.org/licenses/LICENSE-2.0}]

Reply:
👍 it is solved